.eslintrc.json

```diff
@@ -0,0 +1,57 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "airbnb-base",
+        "plugin:@typescript-eslint/recommended",
+        "prettier",
+        "prettier/@typescript-eslint"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint",
+        "prettier"
+    ],
+    "rules": {
+        "prettier/prettier": "error",
+        "semi": ["error", "never"],
+        "lines-between-class-members": ["error", "always", { "exceptAfterSingleLine": true }],
+        "@typescript-eslint/explicit-function-return-type": ["warn", {
+            "allowExpressions": true
+        }],
+        "@typescript-eslint/interface-name-prefix": ["error", "always"],
+        "@typescript-eslint/prefer-interface": "off",
+        "@typescript-eslint/array-type": ["error", "array-simple"]
+    },
+    "overrides": [
+        {
+            "files": ["**/*.spec.ts"],
+            "env": {
+                "jest": true
```

Comment:
Not a suggestion, but a question: why do we need to list jest in the `"overrides"` and not in the `"env"` above?

Reply:
Because jest environment only exist in the context of a test